source/features/hide-own-stars.tsx

```diff
@@ -8,7 +8,7 @@ import {safeElementReady} from '../libs/dom-utils';
 
 const observer = new MutationObserver(([{addedNodes}]) => {
 	// Remove events from dashboard
-	for (const item of select.all('#dashboard .news .watch_started, #dashboard .news .fork')) {
+	for (const item of select.all<HTMLAnchorElement>('#dashboard .news .watch_started, #dashboard .news .fork')) {
```

Comment:
Actually, I think it's potential child is an anchor only. Not sure what this should be.

Reply:
I'd stick to `HTMLElement`